common/global_utils.py

```diff
@@ -0,0 +1,144 @@
+import os
+import json
+import random
+
+from pathlib import Path
+from minio import Minio
+from PIL import Image
+from typing import List
+
+from loguru import logger
+
+
+class EnvFunctionHandler:
+    """
+    Class for managing environment-dependent functions.
+    """
+
+    def __init__(self):
```

Comment:
add return typehint just to keep convention

Reply:
Done